examples/sandbox/index.ts

```diff
@@ -28,10 +30,14 @@ import * as dashTxJson from './json/dashTx.json'
 import * as dogeTxJson from './json/dogeTx.json'
 import * as ltcTxJson from './json/ltcTx.json'
 
+import Portis from "@portis/web3";
+
 const keyring = new Keyring()
 
+const portis = new Portis('ff763d3d-9e34-45a1-81d1-caa39b9c64f9', 'mainnet');
```

Comment:
This should happen inside the `Web3PortisAdapter`. If you need to pass arguments to it, follow the pattern that the `TrezorConnect` one uses.